diff --git a/node_modules/sqlite3/.DS_Store b/node_modules/sqlite3/.DS_Store
new file mode 100644
index 0000000..10f4d43
Binary files /dev/null and b/node_modules/sqlite3/.DS_Store differ
diff --git a/node_modules/sqlite3/lib/sqlite3.js b/node_modules/sqlite3/lib/sqlite3.js
index 430a2b8..9df0857 100644
--- a/node_modules/sqlite3/lib/sqlite3.js
+++ b/node_modules/sqlite3/lib/sqlite3.js
@@ -1,5 +1,5 @@
 const path = require('path');
-const sqlite3 = require('./sqlite3-binding.js');
+const sqlite3 = require('./../build/Release/node_sqlite3.node');
 const EventEmitter = require('events').EventEmitter;
 module.exports = exports = sqlite3;
 
